packages/flutter_tools/test/general.shard/android/gradle_errors_test.dart

```diff
@@ -913,10 +913,10 @@ A problem occurred evaluating project ':app'.
           '│ To fix this issue, replace the following content:                                │\n'
           '│ /android/build.gradle:                                                           │\n'
           "│     - classpath 'com.android.tools.build:gradle:<current-version>'               │\n"
-          "│     + classpath 'com.android.tools.build:gradle:7.3.0'                           │\n"
+          "│     + classpath 'com.android.tools.build:gradle:$templateAndroidGradlePluginVersion'                           │\n"
```

Comment:
~This may seem silly but when I open this file in github neither of these variables exist when I search. Are you sure this works?~ 
Edit: consider named import instead.

Reply:
Done in the file where it is a new import